sdk/fabric/Azure.ResourceManager.Fabric/tests/Scenario/FabricCapacityTests.cs

```diff
@@ -0,0 +1,259 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Models;
+using Azure.ResourceManager.Fabric;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Resources.Models;
+using Azure.ResourceManager.TestFramework;
+using Azure.ResourceManager.Fabric.Models;
+using NUnit.Framework;
+using Microsoft.AspNetCore.Http;
+using System.Net;
+
+namespace Azure.ResourceManager.Fabric.Tests.Scenario
+{
+    public class FabricCapacityTests : FabricManagementTestBase
+    {
+        private FabricCapacityCollection _fabricCapacityCollection { get => ResourceGroup.GetFabricCapacities(); }
+
+        public FabricCapacityTests(bool isAsync)
+            : base(isAsync, RecordedTestMode.Record)
```

Comment:
this record mode is required when you are recording them, please make sure we remove this record mode when you push your code.
```suggestion
            : base(isAsync)//, RecordedTestMode.Record)
```